sdk/react/src/lib/fragment-components/a-definition-tooltip.tsx

```diff
@@ -0,0 +1,221 @@
+		{ id: 'left-top', text: 'Left top' },
+		{ id: 'right', text: 'Right' },
+		{ id: 'right-bottom', text: 'Right bottom' },
+		{ id: 'right-top', text: 'Right top' }
+	];
+
+	return <>
+		<Dropdown
+			id='alignment'
+			label='Alignment the text'
+			titleText='Alignment'
+			items={alignments}
+			selectedItem={alignments.find(item => item.id === selectedComponent.alignment)}
+			itemToString={(item: any) => (item ? item.text : '')}
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				alignment: event.selectedItem.id
+			})} />
+		<TextInput
+			id='description'
+			value={selectedComponent.description}
+			labelText='Description'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				description: event.currentTarget.value
+			})} />
+		<TextInput
+			id='definition'
+			value={selectedComponent.definition}
+			labelText='Tooltip Message'
+			onChange={(event: any) => setComponent({
+				...selectedComponent,
+				definition: event.currentTarget.value
+			})} />
+		<Checkbox
+			id='defaultOpen'
+			labelText='Default open'
+			checked={selectedComponent.isDefaultOpened}
+			onChange={(_: any, { checked }: any) => setComponent({
+				...selectedComponent,
+				isDefaultOpened: checked
+		})} />
+		<Checkbox
+			id='openOnHover'
+			labelText='Open on hover'
+			checked={selectedComponent.isOpenOnHover}
+			onChange={(_: any, { checked }: any) => setComponent({
+				...selectedComponent,
+				isOpenOnHover: checked
+		})} />
+	</>;
+};
+
+export const ADefinitionTooltipCodeUI = ({ selectedComponent, setComponent }: any) => <TextInput
+	value={selectedComponent.codeContext?.name}
+	id='input-name'
+	labelText='Input name'
+	onChange={(event: any) => {
+		setComponent({
+			...selectedComponent,
+			codeContext: {
+				...selectedComponent.codeContext,
+				name: event.currentTarget.value
+			}
+		});
+	}} />;
+
+export const ADefinitionTooltip = ({
+	componentObj,
+	...rest
+}: any) => {
+	return (
+		<AComponent
+		componentObj={componentObj}
+		rejectDrop={true}
+
+		{...rest}>
+			<div className={`${preventCheckEvent} ${componentObj.cssClasses?.map((cc: any) => cc.id).join(' ')} `}>
+				<DefinitionTooltip
+					className={css`${styleObjectToString(componentObj.style)}`}
+					definition={componentObj.definition}
+					align={componentObj.alignment}>
+						{componentObj.description}
+				</DefinitionTooltip>
+			</div>
+		</AComponent>
+	);
+};
+
+export const componentInfo: ComponentInfo = {
+	component: ADefinitionTooltip,
+	settingsUI: ADefinitionTooltipSettingsUI,
+	codeUI: ADefinitionTooltipCodeUI,
+	render: ({ componentObj, select, remove, selected }) => <ADefinitionTooltip
+		componentObj={componentObj}
+		select={select}
+		remove={remove}
+		selected={selected} />,
+	keywords: ['definition tooltip', 'definition', 'tooltip'],
+	name: 'Definition tooltip',
+	type: 'definition-tooltip',
+	defaultComponentObj: {
+		type: 'definition-tooltip',
+		description: 'description'
```

Comment:
if we render children instead of `description`, we can instead add `items` with a text component in here.

_(also, "description" is not a great value for default description)_

Reply:
we do not render children and stick with support only text